hkmc2/shared/src/test/mlscript/basics/ModuleMethods.mls

```diff
@@ -1,115 +1,166 @@
-:js
```

Comment:
Why no JS?

Reply:
This is because module method related stuffs are done completely before the lowering stage. Things that related to runtime (e.g. type classes) are in another test file.

Reply:
Ok, fair enough.

It's just that my default strategy is "always turn in JS to add more coverage to our JS codegen tests", even if the tested feature is about type/module checking.